config/express.js

```diff
@@ -0,0 +1,122 @@
+const log = require('../app/lib/logger');
+const express = require('express');
+const helmet = require('helmet');
+const cookieParser = require('cookie-parser');
+const bodyParser = require('body-parser');
+const compression = require('compression');
+const nunjucks = require('nunjucks');
+const router = require('./routes');
+const locals = require('../app/middleware/locals');
+const constants = require('../app/lib/constants');
+const promBundle = require('../app/lib/promBundle').middleware;
+const errorCounter = require('../app/lib/promCounters').errorPageViews;
+
+module.exports = (app, config) => {
+  // eslint-disable-next-line no-param-reassign
+  app.locals.SITE_ROOT = constants.SITE_ROOT;
+  // eslint-disable-next-line no-param-reassign
+  app.locals.ASSETS_URL = constants.ASSETS_URL;
+  // start collecting default metrics
+  promBundle.promClient.collectDefaultMetrics();
+
+  app.set('views', `${config.root}/app/views`);
+  app.set('view engine', 'nunjucks');
+  const nunjucksEnvironment =
+    nunjucks.configure(`${config.root}/app/views`, {
+      autoescape: true,
+      express: app,
+      watch: true,
+    });
+  log.info({ config: { nunjucksEnvironment } }, 'nunjucks environment configuration');
+
+  app.use(helmet({
+    frameguard: { action: 'deny' },
+    hsts: { includeSubDomains: false },
+    contentSecurityPolicy: {
+      directives: {
+        defaultSrc: [
+          '\'self\'',
+        ],
+        childSrc: [
+          '*.hotjar.com',
+        ],
+        scriptSrc: [
+          '\'self\'',
+          '\'unsafe-eval\'',
```

Comment:
Could this line and the next line be removed? I can't recall whether they are there due how one of the third party scripts works.